pkgs/tools/networking/hue-cli/default.nix

```
@@ -1,10 +1,15 @@
-{ bundlerEnv, ruby }:
+{ lib, bundlerApp }:
 
-bundlerEnv rec {
-  name = "hue-cli-${version}";
-
-  version = (import gemset).hue-cli.version;
-  inherit ruby;
+bundlerApp {
+  pname = "hue-cli";
   gemdir = ./.;
-  gemset = ./gemset.nix;
+  exes = [ "hue" ];
+
+  meta = with lib; {
+    description = "Command line interface for controlling Philips Hue system's lights and bridge.";
```

```suggestion
    description = "Command line interface for controlling Philips Hue system's lights and bridge";
```

> Don’t include a period at the end.

https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes